server/src/main/java/org/elasticsearch/search/SearchService.java

```diff
@@ -133,6 +133,8 @@
     public static final TimeValue NO_TIMEOUT = timeValueMillis(-1);
     public static final Setting<TimeValue> DEFAULT_SEARCH_TIMEOUT_SETTING =
         Setting.timeSetting("search.default_search_timeout", NO_TIMEOUT, Property.Dynamic, Property.NodeScope);
+    public static final Setting<Boolean> DEFAULT_ALLOW_PARTIAL_SEARCH_RESULTS =
+            Setting.boolSetting("search.default_allow_partial_results", true, Property.Dynamic, Property.NodeScope);
```

Comment:
The problem with this being node scope is that we could potentially have different values on different nodes (if the user sets this in `elasticsearch.yml`) which could cause unexpected behaviour. The same is actually already true for the default timeout above but I wonder if we should raise this in an issue?

Reply:
Also, see my comment against the documentation about whether we should make this default to false in this PR to make the backport straightforward and then change the default in a follow up

Reply:
Not sure there is a "Property.ClusterScope"? All cluster-level settings I see (e.g.  RecoverySettings.INDICES_RECOVERY_MAX_BYTES_PER_SEC_SETTING ) come defined with Property.NodeScope.

Reply:
NO unfortunately there is no `Property.ClusterScope` so I think it makes it more important to have the setting resolved on the coordinating node so that all shards use the same value.